src/Array/index_of.js

```diff
@@ -0,0 +1,7 @@
+Array.prototype.myIndexOf = function (searchItem, startIndex = 0) {
+  for(let i = startIndex; i < this.length; i++){
+    if(this[i] === undefined || this[i] === NaN) continue;
```

Comment:
To compare if something is a `NaN`, you need to do `isNaN(this[i])`. `this[i] === NaN` doesn't work.

Reply:
Ah, I see. Thanks!